packages/spelunker-web/src/components/core/GameMap/index.jsx

```diff
@@ -18,24 +19,58 @@ const crs = Object.assign({}, CRS.Simple, {
   infinity: false,
 });
 
+const loadTileIndex = async (tileDirectory) => {
+  // Check index cache
+  if (TILE_INDEX[tileDirectory]) {
+    return;
+  }
+
+  const indexResponse = await fetch(`${process.env.MINIMAP_URI}/tiles/${tileDirectory}`);
+  const tileIndex = await indexResponse.json();
+
+  // Cache index
+  TILE_INDEX[tileDirectory] = tileIndex;
+};
+
 class MinimapTileLayer extends TileLayer {
   getTileUrl({ x, y }) {
     const tx = 32 + x;
     const ty = 32 + y;
-    const mapName = this._url;
-    return `${process.env.PIPELINE_URI}/minimap/${mapName}/${tx}/${ty}.blp.png`;
+
+    // TODO use real url
+    const unknownTileUrl = `${process.env.PIPELINE_URI}/files/textures/minimap/unknown_${tx}_${ty}`;
```

Comment:
Given the TODO here, is this a 404 at present? Could this be a generic URL that does not need `tx` and `ty`?

Reply:
Yeah, this is an intentional 404. The `tx` and `ty` is present just to make it easier to trace while we're monkeying a bunch with `GameMap`.

Do you know if Leaflet accepts `null` responses from `getTileUrl`? I haven't looked too closely yet.